backend/app/database/models.py

```diff
@@ -41,10 +41,11 @@ class SentHistory(Base):
     date_sent = Column(String)
     user_id = Column(Integer, ForeignKey("user.id"))
 
+class ContactList(Base):
+    __tablename__ = "contactlist"
 
-# TODO: Migrate other tables
-# class Email_Sent(Base):
-#     __tablename__ = "email_sent"
-
-#     id = Column(Integer, primary_key=True, index=True)
-#     user_id = Column(Integer, ForeignKey("user.id"),  index=True)
+    id = Column(Integer, primary_key=True, index=True)
+    email = Column(String)
+    user_id = Column(Integer, ForeignKey("user.id"), nullable=False)
+    is_subscribed = Column(Boolean, nullable=False)
```

Comment:
shouldnt the initial value be True instead of False?

On my understanding:
if is_subscribed is True, send email.

Reply:
I think you're right. It should be.
Let me set the initial value to True here.

Reply:
Just an FYI but this part `nullable=False` is banning null on this column. just in case if you thought its the initial value